students/grzegorz_bajorski/lesson_02_flow_control/bishop_move.py

```diff
@@ -0,0 +1,7 @@
+x1 = int(input())
+y1 = int(input())
+x2 = int(input())
+y2 = int(input())
+
+print('YES') if (abs(x1-x2) == abs(y1-y2)) else print('NO')
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.